### ISSUE TITLE
Invalid operation when adding service principal to AD group

### ISSUE THREAD

**Issue by isobit** [Azure/azure-cli#7621]
**Describe the bug**
Attempting to add a service principal to an active directory group results in the following error:

```
An invalid operation was included in the following modified references: 'members'.
```

**To Reproduce**

1. Create an AD security group, get the object ID as `GROUP_ID`
2. Create a service principal, get the object ID as `SP_ID`
3. `az ad group add member --group $GROUP_ID --member-id $SP_ID`

**Expected behavior**
Service principal should be added to AD group.

**Environment summary**
- Install method: pip
- CLI version: 2.0.47
- OS: CentOS 7
- Shell: zsh
- Python version: 3.6

**Comment by mattops** [comment]
Are you sure you used the correct SP_ID?

Should be the `objectId` that is returned by `az ad sp list` - note this is different to the object ID that you see in the portal!

**Comment by isobit** [comment]
Oh, I did not realize those were different things. I used the object ID from `az ad sp list` and that worked just fine. Thanks!

Although it is fairly confusing that both the CLI and the portal have something called an Object ID even though they're different. It would be helpful to include a note in the docs to clarify that... would it make sense to open an issue on [this page](https://docs.microsoft.com/en-us/cli/azure/ad/group/member?view=azure-cli-latest#az-ad-group-member-add)?
